src/webui/api/torrentscontroller.cpp

```diff
@@ -304,6 +312,8 @@ void TorrentsController::trackersAction()
     if (!torrent)
         throw APIError(APIErrorType::NotFound);
 
+    trackerList << getStickyTrackers(torrent);
```

Comment:
`QVariantList trackerList {getStickyTrackers(torrent)};`

Reply:
This change results in the `getStickyTrackers` QList being added as an element of `trackerList`, rather than the elements of `getStickyTrackers` being added.

Reply:
oops, it is because using the braces, can you give it one last try: 
```c++
QVariantList trackerList = getStickyTrackers(torrent);
```

Reply:
That worked!